Sources/Client/Client.cpp

```diff
@@ -519,6 +519,18 @@ namespace spades {
 			}
 		}
```

Comment:
Reset `hasNextSpawnConfig` in `SpawnPressed`. Consider the following scenario:
1. While being alive, the player sets the next spawn configuration C1. `hasNextSpawnConfig` ← `true`
2. The current round concludes, and the game switches to the next map.
3. The player decides an initial spawn configuration C2 and spawns.
4. The player dies.
5. The player spawns with C1 because `hasNextSpawnConfig == true`.

This behavior would be surprising.